triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -152,8 +144,9 @@ object Server {
       maxFailureNumberOfRetries: Int,
       failureRetryTimeRange: Duration,
       dar: Option[Dar[(PackageId, Package)]],
+      jdbcConfig: Option[JdbcConfig],
```

Comment:
I think it's time to have a parameter pack here. The list of parameters just keeps growing. Maybe not in this PR but if it were to be it would be very much in keeping with the rest.